research/src/sources/primer.json5

```diff
@@ -15,69 +15,84 @@
       "concepts": [
         {
           "name": "primary",
-          "category": "type",
           "image": "primer-primary-button.png",
           "$comment": "For form actions and primary page actions."
         },
         {
           "name": "default",
-          "category": "type",
           "image": "primer-default-button.png",
-          "$comment": "Used to indicate the primary action on a page"
+          "$comment": "Used to indicate the primary action on a page."
+        },
+        {
+          "name": "small",
+          "image": "primer-small-button.png",
+          "$comment": "Used to indicate the primary action on a page."
         },
         {
           "name": "danger",
-          "category": "type",
           "image": "primer-danger-button.png",
-          "$comment": "They help reiterate that the intended action is important or potentially dangerous (e.g., deleting a repo or transferring ownership)"
+          "$comment": "They help reiterate that the intended action is important or potentially dangerous (e.g., deleting a repo or transferring ownership)."
         },
         {
           "name": "outline",
-          "category": "type",
           "image": "primer-outline-button.png",
           "$comment": "Outline buttons downplay an action as they appear like boxy links."
         },
         {
           "name": "link",
-          "category": "type",
```

Comment:
If a design system has a variant value that is part of a category of values, like `type="primary"`, then we'll record the variant name as `primary` and the category as `type`.  More on this and why here: https://open-ui.org/contribute/design-system.